tools/scripts/acr-build.ps1

```diff
@@ -79,7 +79,12 @@ else {
         # dev feature builds
         $namespace = $namespace.Replace("feature/", "")
     }
-    elseif ($namespace.StartsWith("release/") -or ($namespace -eq "main") -or ($namespace -eq "master")) {
+    elseif (($namespace -eq "main")) {
```

Comment:
Decision was to leave the 'main' namespace as default and build resides in industrialiotdev container registry.